### PR TITLE
Application summary 

### PR DESCRIPTION
**Pull request by fumimowdan**
-------

## Description
Add an application summary step before submitting the application

## Trello Card Link
[Trello card](https://trello.com/c/QoYr9h0J/88-application-summary-page-placeholder)

### CODE REVIEW

**Comment by fumimowdan** [comment]
[Application summary page (PLACEHOLDER)](https://trello.com/c/QoYr9h0J/88-application-summary-page-placeholder)

File: app/helpers/applicants/application_summary_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Applicants::ApplicationSummaryHelper
```
Comment (raul-gracia) [inline app/helpers/applicants/application_summary_helper.rb:1]: ❓ We probably don't need this file right?

**Review by raul-gracia — APPROVED** [review]
✅ Looks Good to me
👏🏻 Good effort!

Please have a look a the Rspec errors and rubocop.

File: app/helpers/applicants/application_summary_helper.rb (hunk 1 above)
Comment (fumimowdan) [reply app/helpers/applicants/application_summary_helper.rb]: deleted

**Pull request merged**